### DISCUSSION TITLE
How to delete pod logs ?

### DISCUSSION THREAD

**Discussion by navin-rai** [kubernetes-client/csharp#1287] [Q&A]
Is it possible to delete pod logs ?

**Comment by tg123** [comment]
no way to delete logs
but you can move pod around to reset logs

**Comment by behdad088** [comment]
Hi @navin-rai 
you can use the kubectl logs command with the --clear flag. Here's an example:
```
kubectl logs <pod-name> --clear

``` 
This will clear the logs for the specified pod. Note that this command will also clear the logs that are currently being written to the pod
